docs/reference/notificationsconfiguration.md

```diff
@@ -16,6 +16,7 @@ Name | Default | Description
 **Triggers** | [Empty] | Templates are used to generate the notification template message.
 **Services** | [Empty] | Services are used to deliver message.
 **Subscriptions** | [Empty] | Subscriptions contain centrally managed global application subscriptions.
+**Context** | [Empty] | Context is used to define some shared context between all notification templates.
```

Comment:
Can we add a context example Below? Maybe similar to what is upstream - https://argo-cd.readthedocs.io/en/stable/operator-manual/notifications/templates/#defining-user-defined-context

Reply:
Added